packages/html/src/utils/analytics.ts

```diff
@@ -0,0 +1,13 @@
+import {AnalyticsOptions} from "../types";
+
+export const getAnalyticsOptions = (analyticsOptions?: AnalyticsOptions) => {
+    return {
+        ...analyticsOptions && {
```

Comment:
It could be simplified now in the separate function and remove the spreadding: 
` return options ? { trackedAnalytics: ...} : {}`

You could also simplify naming and rename `analyticsOptions` to just `options` as this is understandable in context of the function.